COMMUNITY.md

```diff
@@ -15,18 +15,8 @@ Add Semantic Kernel events to your calendar - we're running two community calls
 * Americas timezone: download the [calendar.ics](https://aka.ms/sk-community-calendar) file.
 * Asia Pacific timezone: download the [calendar-APAC.ics](https://aka.ms/sk-community-calendar-apac) file.
 
-Add Semantic Kernel Development Office Hours for Python and Java to your calendar to help with development:
-* Java Development Office Hours: [Java Development Office Hours](https://aka.ms/sk-java-dev-sync)
-* Python Development Office Hours: [Python Development Office Hours](https://aka.ms/sk-python-dev-sync)
-
 If you have any questions or if you would like to showcase your project(s), please email what you'd like us to cover here: skofficehours[at]microsoft.com.
 
 If you are unable to make it live, all meetings will be recorded and posted online.
 
-## Join the conversation on Discord
-
-We have a growing and active channel on Discord where you can get help, engage in lively discussion,
-and share what you've built with Semantic Kernel!
 
-Join our Discord:
-[https://aka.ms/SKDiscord](https://aka.ms/SKDiscord) 
```

Comment:
Instead of deleting this should we explain what the purpose of Discord is and redirect people to GH discussions if they want to engage with the development team rather than the community?

Reply:
@sophialagerkranspandey  can you take a look at Mark's comment here?